### ISSUE TITLE
Live Alternative Clonezilla.iso UEFI boot option missing

### ISSUE THREAD

**Issue by homza** [stevenshiau/clonezilla#33]
Hi, the new Alternative .iso wont boot via UEFI for me.
The file structure was changed around for some reason anyway it doesn't work for me.    
clonezilla-live-20180329-artful-amd64.iso
EFI/boot/ missing /images folder and efiboot.img
It's now located under boot/grub/efiboot.img 
VS the old file structure of
clonezilla-live-20170905-zesty-amd64.iso
EFI/boot/images/efiboot.img
Boot-able USB created using:
`sudo dd bs=4M if=/home/me/ramdisk/clonezilla-live-20180329-artful-amd64.iso of=/dev/sdb status=progress && sync
`
OS: Arch
Just some user feedback.
I think this the right place to post this, if not I apologize...

**Comment by stevenshiau** [comment]
No idea. It works here. You can try to boot the iso file in a virtual machine, like virtualbox or vmware. Make sure the downloaded iso is flawless.

Steven

**Comment by homza** [comment]
I played around with it some, what I found:

Concern:
If I make a boot-able USB-stick using “DD” and the Clonezilla ISO it does not work as expected with the current version clonezilla-live-20180329-artful-amd64.iso it results in BIOS only boot-able media, no UEFI.
 
Fix:
If I simply format a USB-stick to FAT32 and extract the current version clonezilla-live-20180329-artful-amd64.zip to the USB stick everything works as expected, resulting in BIOS and UEFI boot-able media.

Comments:
I always just downloaded the current alternative ISO and DD it to a USB-stick, did it that way for 5 or so years not really sure why it no longer works that way.
...
iso md5 ..OK
zip method .. Works as expected.
.. Quit Testing ..
user error .. Unknown
clonezilla issue .. Unknown 
uefi vendor firmware issue .. Unknown* 
hardware issue .. Unknown
...

Thank you Steven for you contributions to the project.

Closed…
